custom-dashboards/.applier/group_vars/all.yml

```diff
@@ -1,83 +1,153 @@
+dashboard_namespace: openshift-monitoring
 openshift_cluster_content:
+  - galaxy_requirements: "{{ inventory_dir }}/../.applier/requirements.yml"
   - object: Infrastructure
     content:
+      - name: Namespace
+        file: "{{ inventory_dir }}/../.openshift/namespace/namespace.yml.j2"
+        tags:
+          - infrastructure
+      - name: Grafana - config secret
+        pre_steps:
+          - role: openshift-toolkit/custom-dashboards/mdt-secret-discovery
```

Comment:
@ramius345 rather than run this role twice, you can run the pre-role at the object level. see https://github.com/redhat-cop/operationalizing-openshift-lab/blob/master/.applier/group_vars/seed-hosts/main.yml#L7-L9